### ISSUE TITLE
Prerequisites for System Drive too low

### ISSUE THREAD

**Issue by LucaBoschetto** [openmediavault/openmediavault-docs#131]
**Describe the improvement you'd like**
Update the System Drive size requirements in [prerequisites.rst](https://github.com/openmediavault/openmediavault-docs/blob/master/prerequisites.rst)

**Additional context**
The prerequisites at [prerequisites.rst](https://github.com/openmediavault/openmediavault-docs/blob/master/prerequisites.rst) state that the System Drive should be min. 4 GiB capacity. With this capacity, on a fresh install, when I run the first update, the system runs out of space while installing linux-image-6.1.0-0.deb11.13-amd64 6.1.55-1~bpo11+1:

```
Unpacking linux-image-6.1.0-0.deb11.13-amd64 (6.1.55-1~bpo11+1) ...
dpkg: error processing archive /var/cache/apt/archives/linux-image-6.1.0-0.deb11.13-amd64_6.1.55-1~bpo11+1_amd64.deb (--unpack):
 cannot copy extracted data for './lib/modules/6.1.0-0.deb11.13-amd64/kernel/net/smc/smc.ko' to '/lib/modules/6.1.0-0.deb11.13-amd64/kernel/net/smc/smc.ko.dpkg-new': failed to write (No space left on device)
```

**Comment by ryecoaaron** [comment]
How much ram does your system have?  The docs say the drive should be "but more than the capacity of the RAM".  4GB is possible but not if your system has a lot of ram because the installer will create a larger swap space for larger amounts of ram.  I have a fresh install that only uses 1.7G of space and I had no problem installing updates.

**Comment by lucabosc** [comment]
I read that specification in the docs, and I indeed set up 1 GB of RAM and 4 GB of System Drive, which according to the docs should be the absolute minimum. Still, that is the error that came out upon upgrade.

Did you do it recently? As you can see, the upgrade that makes it fail is the new kernel.

**Comment by ryecoaaron** [comment]
My install is from December.  I even reverted to snap and install two months of updates.  I might not have install with the OMV iso (don't remember).  I have a running cm3588 install that is only using 2.9G.  Maybe it is harder to keep it small with amd64.  I guess I use 16G or larger when creating VMs just to be safe.  Maybe the minimum could stay 4G but a "recommended" could be added.

**Comment by lucabosc** [comment]
I would suggest saying that 4 GB are sufficient for running OMV, but that in case of large upgrades (such as this one) that could not be enough. But still, what's the largest upgrade that could ever be distributed?

**Comment by votdev** [comment]
I started with a 4GiB DOM, but now i'm on 16GiB. 4GiB can be OK, but only when no additional plugins are installed.

@lucabosc What is your recommendation? How much disk space should be the minimum?

**Comment by PrplHaz4** [comment]
I agree the recs are unclear and stumbled on it a bit (and resized multiple times) - maybe say something like (or provide "minimum" and "recommended" separately):
> 8gb + 1GB for every GB of RAM

Things that will consume disk space that will factor in:
- log files
- installing plugins
- installing docker (even if mounted volumes are elsewhere)
- keeping multiple (backup) kernels - at least 1 version prior should be considered required IME
- recovery tools (via omv-extras)

**Comment by mckaygerhard** [comment]
To address this issue I will replicate an installation on a 4GB disk to determine how feasible this environment is. So then we can determinate a way or witch requirements are really need and witch are really possible or not.. 

i post feedback later

**Comment by ryecoaaron** [comment]
4GB might be enough for a fresh system but after the system has been around a while and possibly not patched frequently, 4GB will not be enough especially to update the system.  No need to test.  I have seen this problem many, many times on the forum and in my testing.

**Comment by ryecoaaron** [comment]
No need to make a video. I am well aware of how to make a smaller install. But the packages are not what use most of the space after the system has been running for a while. Trying to use 4G can be done by someone with good linux skills if the system has very few services running. It is too small for most OMV users. It is hard to find storage that small these days and people should not be using old storage.  Feel free to use 4gb in system but the docs should not change.

**Comment by ryecoaaron** [comment]
The OMV always installs all updates from the Update tab or using onv-upgrade. People should be able to upgrade from the web interface.

**Comment by mckaygerhard** [comment]
> The OMV always installs all updates from the Update tab or using onv-upgrade. People should be able to upgrade from the web interface.

umm this maybe coudl be a reason to.. i will leave running the 4G installation to demostrate is valid.. but.. 

if the "Update tab" always install all the updates.. and the download package of the apt/cache fill all available space.. then 4G is not a valid point.. 

i will propose a change that may liked to you all guys.. let me find a way

**Comment by mckaygerhard** [comment]
Ok, i setup 4G installation, but using bootstrap...resiserfs4 kernel and also normal official Debian kernel... both cases for this.. 

>  Trying to use 4G can be done by someone with good linux skills if the system has very few services running. It is too small for most OMV users

4G install are possible for semi-novices.. with minimal knowledge,, using defaults the only **key is not the usage of the web gui for updates.. with simple manual updates over terminal is easy managed**.. 

> the packages are not what use most of the space after the system has been running for a while ...

not so true! i made possible 4G runing system by updated just on each notification..

>  after the system has been around a while and possibly not patched frequently, 

yes so accumulation of packages will be not so big and also ALWAYS CLEARING APT CACHE.. (that can be made also from web UI).. so

### so in conclusion

 is **clearly stated in the documentation that minimum is not the same as recommended**, when we talk about minimum we are talking about the fact that **the person who administers must know or have minimum knowledge, if _they do not have it they should go for the recommended ones_**. 

### my proposal

@ryecoaaron  i expanded the documentation with #141 and i hope main developer approves.. BUT since it is understandable! developers do not understand this behavior to less experienced users ,**BUT the documentation is correct, just a bit rough.**,
